packages/x-components/src/x-modules/next-queries/components/next-queries.vue

```diff
@@ -100,16 +100,52 @@
 You don't need to pass any props, or slots. Simply add the component, and when it has any next
 queries it will show them
 
-```vue
-<NextQueries />
+```vue live
+<template>
+  <SearchInput />
+  <NextQueries />
+</template>
+
+<script>
+  import { SearchInput } from '@empathyco/x-components/search-box';
+  import { NextQueries } from '@empathyco/x-components/next-queries';
+
+  export default {
+    name: 'NextQueriesDemo',
+    components: {
+      SearchInput,
+      NextQueries
+    }
+  };
+</script>
 ```
 
 The component has three optional props. `animation` to render the component with an animation,
 `maxItemToRender` to limit the number of next queries will be rendered (by default it is 5) and
 `highlightCurated` to indicate if the curated Next Queries inside the list should be highlighted.
 
-```vue
-<NextQueries :animation="FadeAndSlide" :maxItemsToRender="10" :highlightCurated="true" />
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work